apps/skilavottord/web/screens/RecyclingCompanies/components/RecyclingCompanyForm/RecyclingCompanyForm.tsx

```diff
@@ -269,6 +333,31 @@ const RecyclingCompanyForm: FC<
               />
             </GridColumn>
           </GridRow>
+          <div
+            style={{
+              display: user?.role && isDeveloper(user.role) ? 'block' : 'none',
+            }}
+          >
+            <GridRow>
+              <GridColumn span="12/12">
+                <Controller
+                  name="isMunicipality"
+                  control={control}
+                  defaultValue={isMunicipality}
+                  render={({ field: { onChange, value, name } }) => (
+                    <Checkbox
+                      large
+                      label="Er sveitarfélag"
+                      name={name}
+                      backgroundColor="blue"
+                      onChange={(e) => onChange(e.target.checked)}
+                      checked={value}
+                    />
+                  )}
+                />
+              </GridColumn>
+            </GridRow>
+          </div>
```

Comment:
The checkbox needs to be in the DOM for all users since the value is being stored in the database. But it is only the developer role who can manipulate it via UI